shared/src/main/scala/coop/rchain/catscontrib/monadOps.scala

```diff
@@ -1,11 +1,19 @@
 package coop.rchain.catscontrib
 
-import ski._
-import cats._, cats.data._, cats.implicits._
+import cats.{Functor, Monad}
+import cats.data.EitherT
+import cats.syntax.all._
+import coop.rchain.catscontrib.ski.kp
 
-final class MonadOps[F[_], A](val self: F[A])(implicit val F: Monad[F]) {
-  def liftM[G[_[_], _]](implicit G: MonadTrans[G]): G[F, A] = G.liftM(self)
-  def forever: F[A]                                         = MonadOps.forever(self)
+// TODO: Rename this syntax to include functor, monad, ... (HKTSyntax?)
+
+final class MonadOps[F[_], A](val self: F[A]) extends AnyVal {
+  def liftM[G[_[_], _]](implicit M: Monad[F], G: MonadTrans[G]): G[F, A] = G.liftM(self)
+  def forever(implicit M: Monad[F]): F[A]                                = MonadOps.forever(self)
+
+  /* EitherT syntax */
+
+  def liftEitherT[L](implicit f: Functor[F]): EitherT[F, L, A] = EitherT.liftF[F, L, A](self)
```

Comment:
This function is useful for list `F[A]` without specifying all type parameters together with F, left and right types.
https://github.com/rchain/rchain/blob/9317f35b5e78550dc5573307d417d28fde6f279c/casper/src/main/scala/coop/rchain/casper/rholang/RuntimeReplaySyntax.scala#L141